modules/dhcp_isc/isc_file_parser.rb

```diff
@@ -7,10 +7,11 @@ module Proxy::DHCP::ISC
   class FileParser
```

Comment:
As mentioned above, I think it's better to keep ipv6 and ipv4 data structures separate. Because of this big chunks of code will move from this class into ipv4/ipv6 specific ones, which might reduce it to a module?